lib/mix/tasks/torch.gen.html.ex

```diff
@@ -27,7 +27,11 @@ defmodule Mix.Tasks.Torch.Gen.Html do
     Enum.each(@commands, &Mix.Torch.inject_templates(&1, format))
 
     # Run the Phoenix generator
-    Mix.Task.run("phx.gen.html", args)
+    if format == "slime" do
```

Comment:
As noted above `format` here should already have converted a config option of `:slim` or `"slim"` into `"slime"` via the `Mix.Torch.parse_config/2` function call above on line 20.